src/components/PriceDeleteConfirmationDialog.vue

```diff
@@ -40,9 +42,26 @@ export default {
   props: {
     'price': null,
   },
+  data() {
+    return {
+      loading: false,
+    }
+  },
   methods: {
     deletePrice() {
-      this.$emit('delete')
+      this.loading = true
+      api
+        .deletePrice(this.price.id)
+        .then((response) => {
+          // if response.status == 204
+          this.loading = false
+          this.removePriceCard()
+          this.$emit('delete')
+          this.closeDialog()
+        })
+    },
+    removePriceCard() {
+      document.getElementById(`price_${this.price.id}`).remove()
```

Comment:
Same issue we had with proofs
![image](https://github.com/openfoodfacts/open-prices-frontend/assets/21193194/c3922197-a486-47ec-aeb4-712248e2e693)
An alternative is to keep in store but find a way to update the store when the database is updated to avoid something like #438

Reply:
indeed, created an issue #449